releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,9 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 3.0.2 - July 01, 2020
+* Fixed the issue that caused a configuration change to be skipped as a part of refresh when a change is detected in a key with `refreshAll` set to `true` and all attempts to retrieve the updated settings from App Configuration fail. [#178](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/178)
+
```

Comment:
Are there other fixes. I remember you mentioned there were other fixes made previously but not released.

Reply:
The only additional change included in this version is to update the user agent header policy to be invoked once per call instead of once per retry. I don't think users care about it, so I have excluded it from the release notes.

Reply:
It's still handy to put down even if it's just for our reference. One thing I've liked about the release notes is being able to pinpoint a version a user on a previous version is using and seeing all of the behavior differences between their version and the latest.

Reply:
One way of doing it is to create milestones for each release, which includes all changes in a release, even a fix not mentioned in release notes as it's not of too much customer interests (but we can always include a link of the milestone in the release notes). This is also beneficial for our planning. For example, we can create a v4.0.0 milestone. We will feel clearer what have done and what is still remaining before we can release it.

Reply:
Milestones aren't a bad idea, but I think they are an added value on top of a complete release notes document. They're kind of a GitHub concept, where as release notes really could be moved anywhere.

As far as mentioning the minor fix, I would include it in the release notes but I'm fine if we don't.

Reply:
We currently have a copy of the release notes at two places - one in this file and the other on the release page [here](https://github.com/Azure/AppConfiguration-DotnetProvider/releases). Currently, both of these are mostly the same, but we can add additional information to one of these if that helps.

Reply:
Let's not complicate this. Keep them the same so it's simpler to maintain. I am fine without mentioning user agent string fix in release notes.